doc/api/async_hooks.md

```diff
@@ -599,11 +604,7 @@ class DBQuery extends AsyncResource {
   }
 
   getInfo(query, callback) {
-    this.db.get(query, (err, data) => {
-      this.emitBefore();
-      callback(err, data);
-      this.emitAfter();
-    });
+    this.db.get(query, this.makeCallback(callback, null, err, data));
```

Comment:
Is this example correct? `err` and `data` do not exist in this scope anymore... I could be missing something though

Reply:
D'oh. Fixed :).